### PR TITLE
Release 0.23 - Thumbs Up

### PR DESCRIPTION
**Pull request by shiftkey**
Aiming to ship this **Thursday 6 October** AEST. Running through the test suite today/tomorrow.

- [x] merge #1486
- [x] merge #1485
- [x] merge #1477
- [x] release notes
- [x] version bump
- [x] integration tests all pass
- [x] :thumbsup: from @Haacked or @ryangribble on release notes 
- [x] tag and publish to NuGet
- [x] :shipit:

### CODE REVIEW

**Comment by ryangribble** [comment]
I'd like to include #1474 if upstream github API fix for create repository call gets resolved before the release ETA

Also I just ran into a minor overlooked thing caused by #1445 today which I've just pushed PR #1485 for, which should be included in the release

File: ReleaseNotes.md [hunk 1]
```diff
@@ -1,3 +1,35 @@
+### New in 0.23.0 (released 06/10/2016)
+
+**Features**
+
+ - Added support for probing for an GitHub Enterprise instance - #1404 via @haacked
```
Comment (Haacked) [inline ReleaseNotes.md]: Maybe this should say "Added support to test whether a URL points to a GitHub Enterprise instance"

File: ReleaseNotes.md [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+**Other**
+
+ - LINQPad samples are now verified at build time - #1456 via @mderriey
+ - More obsolete APIs removed - #1458 via @ryangribble
+ - .NET Core support has been started - #1462 via @mderriey
+
+**Breaking Changes**
+
+Repository identifiers returned from the GitHub API will exceed `Int32.MaxValue` in
+around 12 months, based on current growth. We've decided to update everywhere we
```
Comment (Haacked) [inline ReleaseNotes.md]: Wow. :scream:

**Comment by shiftkey** [comment]
@ryangribble I've added those two PRs to my checklist

**Comment by shiftkey** [comment]
I'll open up some tasks for those integration tests I've muted here tomorrow.

**Comment by ryangribble** [comment]
Are you still shooting for Thursday release or going to go earlier?

Regarding #1474 I checked just now and the behaviour is unchanged.  It's reasonably safe to 🚢 anyway since it just means that MergeMethod field is ignored on repo creation, and the response fields are returned `null` in the response.  Once github.com is rectified, our Create method integration tests should pass.  Meanwhile the `Edit()` `Get()` repository methods work fine, so the workaround is to create a repo then use the `Edit()` method to set the allowed merge methods.  I'd like to get support out for configuring those merge methods if we can, since it's a popular feature and it'd be good to support it this quickly after it was anounced.  Thoughts?

**Comment by shiftkey** [comment]
@ryangribble yeah, still shooting for Thursday. I'll look over #1474 in the morning and check the tests are good to go.

**Comment by ryangribble** [comment]
If the `Create()` method is still no good, we could back out those changes if you aren't  happy to release it in that state while waiting for github API fix.  It will still be useful to release the `Update()` and `Get()` Repository methods since the functionality can totally be exercised with them

**Comment by shiftkey** [comment]
@ryangribble cool, I'll throw some feedback in that PR to see if we should partially ship it

**Comment by ryangribble** [comment]
Turns out it's a doco error and the create method isn't going to support the feature at this time, so I'll fix up that PR tomorrow

**Comment by ryangribble** [comment]
#1477 is good to 🚢

**Comment by shiftkey** [comment]
Oops, missed this yesterday due to other commitments. I'll push this out this afternoon.

**Comment by shiftkey** [comment]
Published:

 - https://www.nuget.org/packages/Octokit/0.23.0
 - https://www.nuget.org/packages/Octokit.Reactive/0.23.0

**Pull request merged**